src/api/app/views/webui2/webui/package/_commit_item.html.haml

```diff
@@ -0,0 +1,35 @@
+- commit = @package.commit(rev.to_s)
+- if commit
```

Comment:
Eat your :spaghetti: please

Reply:
One key learning this week is to decode @hennevogel comments :smile:

Reply:
My comments-decoder machine says:

``
if commit = @package.commit(rev.to_s)  # if you use commit later
``
or

``
if  @package.commit(rev.to_s)  # if you don't use commit anymore
``

Is my machine working properly?

Reply:
It's about getting rid of the spaghetti in my code hahaha